### PR TITLE
Replace `coord` with `location` in `PlaceTypeEnumeration`

### PR DESCRIPTION
**Pull request by ue71603**
Enumeration updated . Fixes https://github.com/VDVde/OJP/issues/168

### CODE REVIEW

**Review by herlitze — DISMISSED** [review]

File: OJP/OJP_Places.xsd [hunk 1]
```diff
@@ -203,10 +203,23 @@
...
+			</xs:enumeration>
 			<xs:enumeration value="address"/>
 			<xs:enumeration value="poi"/>
-			<xs:enumeration value="coord"/>
+			<xs:enumeration value="coord">
+				<xs:annotation>
+					<xs:documentation>deprecated. Is replaced by geoPosition.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="geoPosition">
```
Comment (skinkie) [inline OJP/OJP_Places.xsd:218]: Wouldn't the transmodel "location" not be a better suited value here?
```suggestion
			<xs:enumeration value="location">
```

File: OJP/OJP_Places.xsd [hunk 2]
```diff
@@ -203,10 +203,23 @@
...
+			</xs:enumeration>
 			<xs:enumeration value="address"/>
 			<xs:enumeration value="poi"/>
-			<xs:enumeration value="coord"/>
+			<xs:enumeration value="coord">
+				<xs:annotation>
+					<xs:documentation>deprecated. Is replaced by geoPosition.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="location">
```
Comment (Aurige) [inline OJP/OJP_Places.xsd:218]: Is an address a location ? I guess yes, but may then be explicit. If not "address" needs to be added

Comment (ue71603) [reply OJP/OJP_Places.xsd:218]: @Aurige Where would we have to add it? In Transmodel? the problem is, that it all should e a PLACE not a LOCATION. An Address is a sort of PLACE: So I guess that what we are doing here, is ok. Or should we change something?

Comment (Aurige) [reply OJP/OJP_Places.xsd:218]: My fault, nothing to add, _address_ is just above... I don't know how I missed it, sorry

File: OJP/OJP_Places.xsd [hunk 3]
```diff
@@ -203,10 +203,23 @@
...
+				<xs:annotation>
+					<xs:documentation>stop covers StopPlace and StopPoint.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 			<xs:enumeration value="address"/>
 			<xs:enumeration value="poi"/>
-			<xs:enumeration value="coord"/>
+			<xs:enumeration value="coord">
+				<xs:annotation>
+					<xs:documentation>deprecated. Is replaced by geoPosition.</xs:documentation>
```
Comment (sgrossberndt) [inline OJP/OJP_Places.xsd:215]: `coord` is replaced by `geoPosition`, but there is no `geoPosition` in the `PlaceTypeEnumeration`.

What about `Deprecated. Use location instead.`

File: OJP/OJP_Places.xsd [hunk 4]
```diff
@@ -203,10 +203,23 @@
...
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="location">
+				<xs:annotation>
+					<xs:documentation>New value for geoPosition.</xs:documentation>
```
Comment (sgrossberndt) [inline OJP/OJP_Places.xsd:220]: What about `Geographic position consisting of coordinates. Replaces the deprecated value coord.`

**Review by sgrossberndt — CHANGES_REQUESTED** [review]
Please try to add more meaningful commit messages so it is easier to identify which changes are actually done in the commit.

File: OJP/OJP_Places.xsd (hunk 3 above)
Comment (ue71603) [reply OJP/OJP_Places.xsd]: fixed

File: OJP/OJP_Places.xsd (hunk 4 above)
Comment (ue71603) [reply OJP/OJP_Places.xsd]: fixed

**Review by Aurige — APPROVED** [review]

**Review by sgrossberndt — APPROVED** [review]

**Comment by trurlurl** [comment]
Documentation updated.

**Review by sgrossberndt — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Review by herlitze — APPROVED** [review]

**Pull request merged**